ambry-store/src/test/java/com.github.ambry.store/BlobStoreTest.java

```diff
@@ -1102,4 +1160,23 @@ private void verifyOperationFailuresOnInactiveStore(BlobStore blobStore) {
       assertEquals("Unexpected StoreErrorCode", StoreErrorCodes.Store_Not_Started, e.getErrorCode());
     }
   }
+
+  /**
+   * Attempt to write a set of messages with keys and crcs from the given lists of keys and crcs.
+   * @param mockIdList the list of keys of the messages.
+   * @param crcList the list of crcs of the messages.
+   * @throws StoreException
+   */
+  private void attemptPut(List<StoreKey> mockIdList, List<Long> crcList) throws StoreException {
```

Comment:
Since there is a `put` function, could you name this differently to better reflect what it is doing? Something like `putWithSpecificCrcs()` or something like that.